statannotations/stats/StatTest.py

```diff
@@ -111,6 +111,14 @@ def short_name(self):
     't-test_ind':       StatTest(stats.ttest_ind,
                                  't-test independent samples',
                                  't-test_ind', 't'),
+    
+    't-test_paired-gt':    StatTest(stats.ttest_rel,
+                                 't-test paired samples', 't-test_rel', 't',
+                                 alternative="greater"),
+
+    't-test_paired-ls':    StatTest(stats.ttest_rel,
+                                 't-test paired samples', 't-test_rel', 't',
+                                 alternative="less"),
```

Comment:
Could you move these below the two-sided counterpart and fix the alignment of the 2nd and 3rd lines?